services/storage/server.js

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env node
+'use strict';
+
+require('dotenv').config();
+
+const boltzmann = require('boltzmann');
```

Comment:
I'd be happy to put in a PR to pull out some of this boilerplate, it'll probably become pretty common.